### DISCUSSION TITLE
3.12.0

### DISCUSSION THREAD

**Discussion by sashirestela** [sashirestela/simple-openai#245] [Announcements]
## What's Changed
* Add Session Tokens API by @sashirestela in https://github.com/sashirestela/simple-openai/pull/240
* Adding reasoningEffort and developer role to ChatCompletions by @sashirestela in https://github.com/sashirestela/simple-openai/pull/241
* Updating FineTuning API including DPO by @sashirestela in https://github.com/sashirestela/simple-openai/pull/242
* Updating Realtime API by @sashirestela in https://github.com/sashirestela/simple-openai/pull/244

**Full Changelog**: https://github.com/sashirestela/simple-openai/compare/v3.11.0...v3.12.0

<hr /><em>This discussion was created from the release <a href='https://github.com/sashirestela/simple-openai/releases/tag/v3.12.0'>3.12.0</a>.</em>

**Comment by Lall-091** [comment]
- - [ ] 
_****_
